src/utils/format-numbers.js

```diff
@@ -131,3 +131,12 @@ export function formatValueTotal(
 
   return value_total + percent;
 }
+
+/**
+ * Format bytes exclusively as megabytes to a certain precision.
+ * e.g. 12345 => "+12.35mb"
+ */
+export function formatBytesAsMegabytes(bytes: number, precision: number = 2) {
+  const mb = bytes / 1024 / 1024;
+  return `${parseFloat(mb.toFixed(precision))}mb`;
+}
```

Comment:
actually I wonder why you don't want to reuse `formatBytes` that we already have above ? :-)

Otherwhise megabytes are `MB`, not `mb` (yeah, I know you're not used to the metric system :p)

Reply:
Please no snarky comments.

Reply:
I switched to `formatBytes`, I think initially I thought megabytes were the most relevant order of magnitude, but who am I to say? This function was written in haste during prototyping anyway.